packages/esm-patient-chart-app/src/visit-header/visit-queue-details.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import VisitQueueDetail from './visit-queue-details.component';
+import { STATUS, PRIORITY } from '../constants';
+import { showModal } from '@openmrs/esm-framework';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  showModal: jest.fn(),
+}));
+
+describe('VisitQueueDetail', () => {
+  const mockQueueEntry = {
+    service: 'Test Service',
+    status: STATUS.IN_SERVICE,
+  };
+
+  const mockPriority = PRIORITY.EMERGENCY;
+
+  it('renders without crashing', () => {
+    const { container } = render(<VisitQueueDetail queueEntry={mockQueueEntry} priority={mockPriority} />);
```

Comment:
Nit: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-container-to-query-for-elements